connection.go

```diff
@@ -122,6 +126,7 @@ func (sc *snowflakeConn) exec(
 	sc.QueryID = data.Data.QueryID
 	sc.SQLState = data.Data.SQLState
 	sc.populateSessionParameters(data.Data.Parameters)
+	sc.startHeartBeat()
```

Comment:
Does this start heartbeating after a successful connection is made?

Reply:
Yes, it is